src/Bridge/Anthropic/ModelHandler.php

```diff
@@ -55,13 +61,28 @@ public function request(Model $model, object|array|string $input, array $options
         ]);
     }
 
+    /**
+     * @throws TransportExceptionInterface
+     * @throws ServerExceptionInterface
+     * @throws RedirectionExceptionInterface
+     * @throws DecodingExceptionInterface
+     * @throws ClientExceptionInterface
+     */
```

Comment:
please remove this again. this is only needed due to PhpStorm inspections, not because of adding value. if you want to introduce valuable error information